src/internal/util/ArgumentOutOfRangeError.ts

```diff
@@ -17,8 +17,6 @@ export interface ArgumentOutOfRangeErrorCtor {
  * @see {@link elementAt}
  * @see {@link take}
  * @see {@link takeLast}
- *
- * @class ArgumentOutOfRangeError
```

Comment:
Maybe you should wait with this file until the transition to es2015+?

Reply:
Not sure why? 🙂 The reason why I'm removing these tag definitions is because they don't bring any value. At first, it may seem like they do, but they don't.

For example, the value from the `@see` tag definition two lines above is used [here](https://github.com/ReactiveX/rxjs/blob/f04fe72eba2f117041e1a9c1794f16473ecae697/docs_app/tools/transforms/templates/api/includes/see-also.html). Also, someone reading the code using an editor/IDE (or even Github) could use this information to _see_ `elementAt` and/or `take`/`takeLast`.

Therefore, there are two ways these tag definitions bring value: to the docs app and to the user. `@class` is not being used anywhere: docs site does not use this info from this tag def* and users reading this wouldn't benefit much (the [code below this tag def](https://github.com/ReactiveX/rxjs/pull/7227/files#diff-e696f1c182b04fde2eaf96783e6489b2ab52a66eaedfc2e534c90d1a1efa73c2L23) clearly indicates an object of the `ArgumentOutOfRangeErrorCtor` type).

The third issue is maintainability: renaming stuff (which was done in this library a lot of times) often leaves docs content outdated. Removing stuff like this removes the need to update docs in multiple places which lowers the possibility of having the outdated content.

*The docs app uses the `dgeni` library to [infer](https://github.com/angular/dgeni-packages/blob/4fac18fb8e03265718a11b52cdcb05e72e9fba0c/typescript/src/api-doc-types/ClassExportDoc.ts#L14) the `docType` and to display it [here](https://github.com/ReactiveX/rxjs/blob/f04fe72eba2f117041e1a9c1794f16473ecae697/docs_app/tools/transforms/templates/api/includes/class-overview.html#L5).